src/components/dialogs/network-modifications/static-var-compensator/creation/susceptance-area.tsx

```diff
@@ -0,0 +1,154 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import Grid from '@mui/material/Grid';
+import {
+    B0,
+    MAX_Q_AT_NOMINAL_V,
+    MAX_SUSCEPTANCE,
+    MIN_Q_AT_NOMINAL_V,
+    MIN_SUSCEPTANCE,
+    Q0,
+    SLIDER_Q_NOMINAL,
+    SLIDER_SUSCEPTANCE,
+} from 'components/utils/field-constants';
+import { FloatInput, SliderInput, TextInput } from '@gridsuite/commons-ui';
+import { gridItem, ReactivePowerAdornment, SusceptanceAdornment } from '../../../dialogUtils';
+import React, { FunctionComponent, useEffect } from 'react';
+import { useFormContext } from 'react-hook-form';
+import { getFloatNumber } from './stand-by-automaton-form-utils';
+import { isValidPercentage } from '../../../percentage-area/percentage-area-utils';
+
+export interface SusceptanceAreaProps {
+    isSusceptance: boolean;
+    isQFixe: boolean;
+    min: number;
+    max: number;
+}
+export const SusceptanceArea: FunctionComponent<SusceptanceAreaProps> = ({ isSusceptance, isQFixe, min, max }) => {
+    const { setValue } = useFormContext();
+
+    useEffect(() => {
+        let avgValue = (Number(min) + Number(max)) / 2;
+        if (isSusceptance) {
+            setValue(B0, avgValue);
+            setValue(SLIDER_SUSCEPTANCE, avgValue);
+        } else if (isQFixe) {
+            setValue(Q0, avgValue);
+            setValue(SLIDER_Q_NOMINAL, avgValue);
+        }
+    }, [setValue, isSusceptance, isQFixe, min, max]);
+
+    const onSliderSusceptanceChange = (value: any) => {
+        setValue(B0, getFloatNumber(value));
+        return value;
+    };
+
+    const onSliderQnomChange = (value: any) => {
+        setValue(Q0, getFloatNumber(value));
+        return value;
+    };
+
+    const handleSusceptanceValueChange = (value: any) => {
+        setValue(SLIDER_SUSCEPTANCE, getFloatNumber(value));
+        return value;
+    };
+
+    const handleQnomValueChange = (value: any) => {
+        setValue(SLIDER_Q_NOMINAL, getFloatNumber(value));
+        return value;
+    };
+
+    const minSusceptanceField = (
+        <FloatInput
```

Comment:
value should be propagated from the parent component, avoid to use rhf component